app/src/main/java/org/simple/clinic/mobius/MobiusActivityDelegate.kt

```diff
@@ -0,0 +1,110 @@
+package org.simple.clinic.mobius
+
+import android.os.Bundle
+import android.os.Parcelable
+import com.spotify.mobius.Connectable
+import com.spotify.mobius.Connection
+import com.spotify.mobius.First.first
+import com.spotify.mobius.Init
+import com.spotify.mobius.MobiusLoop
+import com.spotify.mobius.Update
+import com.spotify.mobius.android.MobiusAndroid
+import com.spotify.mobius.extras.Connectables
+import com.spotify.mobius.functions.Consumer
+import com.spotify.mobius.functions.Function
+import com.spotify.mobius.rx2.RxMobius
+import io.reactivex.Observable
+import io.reactivex.ObservableTransformer
+import io.reactivex.disposables.Disposable
+import org.simple.clinic.crash.CrashReporter
+import org.simple.clinic.util.unsafeLazy
+
+class MobiusActivityDelegate<M : Parcelable, E, F>(
```

Comment:
I am assuming that this is the same as `MobiusDelegate`, but with only the changes to the `onSaveInstanceState` and `onRestoreInstanceState` functions to accommodate the differences for an `Activity`.

Reply:
Do we plan to unify the two classes later once we migrate one activity to Mobius?

Reply:
There are some changes made to the start function as well and prepare is no longer exposed. This is due to subtle differences between the lifecycle of the `View` and the `Activity`.

Reply:
Alright, let's discuss unifying them later.